torchci/lib/bot/rerunGithubInfraErrorWorkflow.ts

```diff
@@ -9,9 +9,9 @@ function rerunGithubInfraErrorWorkflow(app: Probot): void {
   // This bot is used to rerun failed workflows on pytorch/pytorch that look
   // like https://github.com/pytorch/pytorch/actions/runs/8454565307
   app.on("workflow_run.requested", async (ctx) => {
-    if (day > new Date().getDate()) {
+    if (day > new Date().getDay()) {
```

Comment:
Seems like day would eventually become 6 (or 7?) and then stay at that value forever, without the if condition ever being true again?